ci/test/pipeline.template.yml

```diff
@@ -378,17 +378,32 @@ steps:
         agents:
           queue: linux-aarch64-small
 
-  - id: ssh-connection
-    label: SSH connection tests
-    depends_on: build-aarch64
-    timeout_in_minutes: 40
-    inputs: [test/ssh-connection]
-    artifact_paths: junit_*.xml
-    plugins:
-      - ./ci/plugins/mzcompose:
-          composition: ssh-connection
-    agents:
-      queue: linux-aarch64
+  - group: "Connection tests"
```

Comment:
Totally up to the testing folks if we want to group these tests. It seemed like a decent idea but I'll 100% defer to y'all!

Reply:
Thanks, the grouping seems fine.